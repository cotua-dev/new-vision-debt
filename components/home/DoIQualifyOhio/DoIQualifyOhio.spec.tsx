import { render, RenderResult } from '@testing-library/react';
import { DoIQualifyOhio } from './index';

describe('DoIQualifyOhio Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<DoIQualifyOhio/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
