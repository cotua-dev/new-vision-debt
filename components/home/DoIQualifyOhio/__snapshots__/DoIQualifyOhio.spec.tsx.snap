// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoIQualifyOhio Component renders 1`] = `
<section
  class="do-i-qualify"
>
  <div
    class="qualify-content"
  >
    <h2
      class="qualify-title"
    >
      Do I Qualify for Debt Consolidation in Ohio?
    </h2>
    <strong
      class="qualify-upper"
    >
      You have over $10,000 in debt
    </strong>
    <p
      class="qualify-lower"
    >
      In Ohio, this is the baseline for us to negotiate on your behalf; it gives us greater leverage and a better chance at settling debts with your creditors.
    </p>
    <strong
      class="qualify-upper"
    >
      You have unsecured debt
    </strong>
    <p
      class="qualify-lower"
    >
      We only work with unsecured debts, which are debts that do not have collateral attached to them. Credit cards, medical bills and payday loans are all examples of unsecured debt. Debts like student loans and those that do have collateral attached to them like mortgages and car loans are not eligible for debt settlement.
    </p>
    <strong
      class="qualify-upper"
    >
      You are able to make monthly payments
    </strong>
    <p
      class="qualify-lower"
    >
      In Ohio, The programs we recommend to you involve involves making deposits of an agreed upon amount into a dedicated account every month â€“ consistent monthly deposits are crucial to your overall success.
    </p>
  </div>
</section>
`;
